### PR TITLE
Remove `phdi` from `message-parser` and use `dibbs` image in Dockerfile

### LINKED ISSUE #1605: remove `phdi` from `message-parser` and use the `dibbs` container on `ghcr.io`
## Action Requested
Remove the `phdi` SDK from the `message-parser` application and use the `dibbs` container on `ghcr`. We will be duplicating code across the codebase temporarily so that we can plant one foot in having our containers be independent of the SDK while keeping our other foot maintaining the SDK as it currently exits.

## Acceptance Criteria
When `phdi` is removed from `message-parser` and the Dockerfile is using the `dibbs` container image from `ghcr`.

### PR DESCRIPTION
**Pull request by robertmitchellv**
# PULL REQUEST

## Summary
Removing the `phdi` SDK functions from the `message-parser` and changing the base image of the Dockerfile to the `dibbs` container hosted on `ghcr`. 

We're using a similar structure to the SDK in terms of where we expect to find and load the functions in our `import` statements. Here is a quick look at the tree structure for relevant areas:

```
.
├── app
│   ├── cloud
│   ├── config.py
│   ├── custom_schemas
│   ├── default_schemas
│   ├── fhir
│   ├── __init__.py
│   ├── main.py
│   ├── models.py
│   ├── phdc
│   ├── __pycache__
│   ├── transport
│   └── utils.py
├── assets
├── description.md
├── dev-requirements.txt
├── Dockerfile
├── generate-test-data
├── requirements.txt
├── schema
└── tests
    ├── assets
    ├── conftest.py
    ├── __init__.py
    ├── integration
    ├── __pycache__
    ├── test_cloud.py
    ├── test_fhir_to_phdc_endpoint.py
    ├── test_fhir_transport.py
    ├── test_parse_message_endpoint.py
    ├── test_phdc.py
    ├── test_phdc_validation.py
    ├── test_schemas_endpoint.py
    ├── test_transport.py
    └── test_utils.py
```

Additionally, tests for the newly folded in parts of the SDK are tested separately based on their module. `README.md` files were also added with a note about where the functions are coming from and a `TODO` to address the move away from the SDK, which we should add to our repo in the future.

## Related Issue
Fixes #1605 
- #1605 

## Checklist

- [x] If this code affects the other scrum team, have they been notified? (In Slack, as reviewers, etc.)
  - [x] Reviewers
  - [x] Slack

[//]: # (PR title: Remember to name your PR descriptively!)

### CODE REVIEW

File: containers/message-parser/requirements.txt [hunk 1]
```diff
@@ -1,6 +1,14 @@
...
 frozendict
-phdi @ git+https://github.com/CDCgov/phdi@main
-uvicorn
+google-auth
+google-cloud-storage
 httpx
+lxml
+requests
+urllib3
+uvicorn
```
Comment (nickclyde) [inline containers/message-parser/requirements.txt:14]: I think you can remove uvicorn as it's included in the base image!

**Review by nickclyde — APPROVED** [review]
Looks great!

File: containers/message-parser/requirements.txt (hunk 1 above)
Comment (robertmitchellv) [reply containers/message-parser/requirements.txt:14]: ah right good catch!

**Review by DanPaseltiner — APPROVED** [review]
Reviewed yesterday, but didn't leave a review.

**Pull request merged**